spec/ic0.txt

```diff
@@ -62,6 +62,7 @@ ic0.data_certificate_copy : (dst: i32, offset: i32, size: i32) -> ();       // *
 ic0.time : () -> (timestamp : i64);                                         // *
 ic0.global_timer_set : (timestamp : i64) -> i64;                            // I G U Ry Rt C T
 ic0.performance_counter : (counter_type : i32) -> (counter : i64);          // * s
+ic0.is_controller: (src: i32, size: i32) -> ( result: i32);                 // * s
```

Comment:
Is it really gonna be available in the WASM _start_ function? @dsarlis

Reply:
I was thinking it's a general system api that can be useful (similar to performance counter or retrieving your balance) so I didn't see a reason not to. I don't feel strongly though, if you believe we don't need it, we can drop start here.

Reply:
I'm fine either way, too, just wanted to make sure it's a deliberate choice.